controller/src/main/java/io/pravega/controller/store/stream/ZKStoreHelper.java

```diff
@@ -28,7 +30,10 @@
 
 @Slf4j
 public class ZKStoreHelper {
-
+    static final String BUCKET_ROOT_PATH = "/buckets";
+    static final String BUCKET_OWNERSHIP_PATH = BUCKET_ROOT_PATH + "/ownership";
```

Comment:
we should probably embed in the path what this ownership is about. Just `ownership/` is ambiguous.

Reply:
this ownership is on buckets. 
the whole path is "buckets/ownership"
individual ephemeral bucket nodes are created under this with respective bucket ids.
buckets/ownership/<bucketid>

Reply:
Ok, I missed that.